client/components/AddTestPlanToQueue/AddTestPlanToQueueContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+
+const AddTestPlanToQueueContainer = ({ handleOpenDialog = () => {} }) => {
```

Comment:
Performance wise, it's not much of a problem and it would be easier for persons to see the types and defaults in a single place. However, the React team has plans to deprecate the use of defaultProps for **functional** components here: https://github.com/facebook/react/pull/16210 with the ES6 standard being preferred which is why I forego its usage. You can read more on the motivation behind that here as well: https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md

Reply:
Ah, thank you for that info, I had 0 idea. Good thing you noticed so we didn't get bit in the behind later.